MdeModulePkg/MdeModulePkg.dsc

```diff
@@ -120,6 +120,8 @@
 
   MmuLib|MdePkg/Library/BaseMmuLibNull/BaseMmuLibNull.inf       ## MU_CHANGE
 
+  PanicLib|MdePkg/Library/BasePanicLibSerialPort/BasePanicLibSerialPort.inf                                        # MU_CHANGE

```

Comment:
Was there a reason to use `BasePanicLibSerialPort` in MdeModulePkg.dsc and UefiCpuPkg.dsc instead of `BasePanicLibNull`?

Normally, the NULL instance will be used for generic builds to reduce dependencies and the functional version used when the code is integrated into a platform.

Reply:
No real reason.  I just forgot convention.  Both `MdeModulePkg` and `UefiCpuPkg` are using the NULL instance now.